gengo/src/documentation.rs

```diff
@@ -9,7 +9,8 @@ impl Documentation {
     }
 
     fn is_documentation_no_read(filepath: &OsStr) -> bool {
-        filepath.to_str().unwrap_or_default().starts_with("docs/")
+        let prefix = format!("docs{}", MAIN_SEPARATOR);
+        filepath.to_str().unwrap_or_default().starts_with(&prefix)
```

Comment:
So I was actually being a bit lazy here by checking if the path starts with `docs/`, focusing more on getting the API set up than accuracy 😅 

This definitely does make it work on Windows now, where before it wouldn't, but I'm wondering if it would be better build a [`Path` and iterate over its components](https://doc.rust-lang.org/stable/std/path/struct.Path.html#method.components) to find `docs`. Or use a regex to check if the path matches `(?:^|/|\\)docs(?:/|\\)`.

Reply:
Good point! I guess I was being lazy too 😅 

I've also added some tests 👍 .